netsim/defaults/hints.yml

```diff
@@ -15,15 +15,22 @@ evpn:
 bgp:
   igp: >
     Add a supported IGP (ospf, isis, eigrp) to the list of modules.
+
 report:
   source: >
     A report can be specified in a file with .j2 suffix within 'reports' subdirectory in
     package-, system-, user- or current directory. You can also specify a report in a
     defaults.outputs.report setting.
+
 quirks:
   junos_lb: >
     Junos devices cannot have more than one loopback interface per routing instance
 
+vlan:
+  fwd_mode_check: >
+    You can disable this error with "defaults.vlan.warnings.mixed_fwd: False", but then
```

Comment:
It may be confusing to have a "fwd_mode_check" that can be disabled through "mixed_fwd"

Suggestion: Either use ```mixed_fwd_check``` or ```fwd_mode_check``` for both?